### ISSUE TITLE
for-all-boosters.sh closes terminal after running -h

### ISSUE THREAD

**Issue by lincolnthree** [snowdrop/booster-common#92]
Should probably leave the terminal open instead :)

**Comment by lincolnthree** [comment]
running like this causes the issue for me:

```
$ . for-all-boosters.sh -h
```

running like this works fine:

```
$ ./for-all-boosters.sh -h
```

**Comment by geoand** [comment]
@lincolnthree I have never seen the `. script` syntax :). Is that something that is common?

**Comment by metacosm** [comment]
I've added a remark on the README that the script is meant to be run and not sourced. I tried to make changes (i.e. use `return` instead of `exit`) but it didn't seem to matter and I don't particularly want to invest time trying to figure this out (though, @lincolnthree, feel free to try to get it to work like you'd expect).

**Comment by lincolnthree** [comment]
Yeah it's not a big deal, but it's the first thing that happened when I tried to run it, and it's always a bit surprising when your terminal window vanishes :) 

@geoand It's a fairly common method of running scripts yes. But... obviously not the only method.

**Comment by geoand** [comment]
@lincolnthree I never came across it, good to know :P

**Comment by metacosm** [comment]
As far as I know it's needed when you don't have the shebang at the beginning of the script. I've rarely done it myself either. @lincolnthree ok to close this issue or do you want to look into it?
